apps/ejabberd/src/mod_commands.erl

```diff
@@ -252,7 +347,61 @@ lookup_recent_messages(ArcJID, WithJID, Before, Limit) ->
     R = ejabberd_hooks:run_fold(mam_lookup_messages, Host, {ok, {0, 0, []}},
                                 [Host, ArcID, ArcJID, RSM, Borders,
                                  Start, End, Now, WithJID, SearchText,
-                                 PageSize, LimitPassed, MaxResultLimit, IsSimple]),
+                                 PageSize, LimitPassed, MaxResultLimit,
+                                 IsSimple]),
     {ok, {_, _, L}} = R,
     L.
 
+create_acc(CallerJid, Name, Type) ->
+    A = mongoose_acc:new(),
+    Map = #{name => Name, type => Type, from => jid:to_binary(CallerJid),
+            from_jid => CallerJid},
+    mongoose_acc:update(A, Map).
+
+create_acc(CallerJid, Name, Type, OtherJid) ->
+    A = create_acc(CallerJid, Name, Type),
+    A1 = mongoose_acc:put(to, jid:to_binary(OtherJid), A),
+    mongoose_acc:put(to_jid, OtherJid, A1).
+
+
+subscription(Caller, Other, Action) ->
+    _ = {invite, accept},
```

Comment:
Nay, it a leftover; it was here to guarantee those atoms exists, but now it is redundant, as well as a few other lines, thanks for pointing it out.